packages/cli/templates/default-backend-module/src/module.ts.hbs

```diff
@@ -0,0 +1,14 @@
+import { coreServices, createBackendModule } from '@backstage/backend-plugin-api';
+
+export const {{moduleVar}} = createBackendModule({
+  pluginId: "{{pluginId}}",
```

Comment:
maybe run prettier on this if that even works (super tiny nit about double quotes used)

Reply:
It is run after templating, but yeah might as well keep things consistent across the templates.